{
	"candidate": "CVE-2017-13747",
	"description": "There is a reachable assertion abort in the function jpc_floorlog2() in jpc/jpc_math.c in JasPer 2.0.12 that will lead to a remote denial of service attack.",
	"notes": {
		"mdeslaur": "introduced in 2.0.12 by https://github.com/mdadams/jasper/commit/a712a2041085e7cd5f2b153e1532ac2a2954ffaa"
	},
	"priority": "low",
	"cvss": {
		"nvd": " CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H [7.5 HIGH]"
	},
	"bugs": [
		"https://bugzilla.redhat.com/show_bug.cgi?id=1485282",
		"https://github.com/mdadams/jasper/issues/71"
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2017-13747"
	],
	"public_date": "2017-08-29 06:29:00 UTC",
	"packages": {
		"artful": {
			"jasper": {
				"name": "jasper",
				"status": "DNE"
			}
		},
		"bionic": {
			"jasper": {
				"name": "jasper",
				"status": "DNE"
			}
		},
		"devel": {
			"jasper": {
				"name": "jasper",
				"status": "DNE"
			}
		},
		"esm-infra-legacy/trusty": {
			"jasper": {
				"name": "jasper",
				"status": "DNE",
				"note": "trusty/esm was DNE [trusty was not-affected [code not present]]"
			}
		},
		"esm-infra/xenial": {
			"jasper": {
				"name": "jasper",
				"status": "not-affected",
				"note": "code not present"
			}
		},
		"precise/esm": {
			"jasper": {
				"name": "jasper",
				"status": "DNE"
			}
		},
		"trusty": {
			"jasper": {
				"name": "jasper",
				"status": "not-affected",
				"note": "code not present"
			}
		},
		"trusty/esm": {
			"jasper": {
				"name": "jasper",
				"status": "DNE",
				"note": "trusty was not-affected [code not present]"
			}
		},
		"upstream": {
			"jasper": {
				"name": "jasper",
				"status": "needs-triage"
			}
		},
		"vivid/ubuntu-core": {
			"jasper": {
				"name": "jasper",
				"status": "DNE"
			}
		},
		"xenial": {
			"jasper": {
				"name": "jasper",
				"status": "not-affected",
				"note": "code not present"
			}
		},
		"zesty": {
			"jasper": {
				"name": "jasper",
				"status": "DNE"
			}
		}
	}
}
